src/main/groovy/org/owasp/dependencycheck/gradle/tasks/AbstractAnalyze.groovy

```diff
@@ -377,18 +377,15 @@ abstract class AbstractAnalyze extends ConfiguredTask {
      * Checks whether a configuration is considered to be a test configuration in order to skip it.
      * A configuration is considered a test configuration if and only if any of the following conditions holds:
      * <ul>
-     *     <li>the name of the configuration or any of its parent configurations equals 'testCompile'</li>
-     *     <li>the name of the configuration or any of its parent configurations equals 'testImplementation'</li>
-     *     <li>the name of the configuration or any of its parent configurations equals 'androidTestCompile'</li>
-     *     <li>the configuration name starts with 'test'</li>
-     *     <li>the configuration name starts with 'androidTest'</li>
+     *     <li>the name of the configuration or any of its parent configurations matches /^(.*[a-z0-9_]T|_?t)est([A-Z0-9_].*)?$/</li>
```

Comment:
Don't forget changing to the new regex here. :-)

Reply:
Also "matches" is no longer correct

Reply:
```suggestion
     *     <li>the name of the configuration or any of its parent configurations matches /((^|[a-z0-9_])T|(^|_)t)est([A-Z0-9_]|$)/</li>
```